neuralprophet/plot_model_parameters_plotly.py

```diff
@@ -0,0 +1,669 @@
+            1 shifts by 1 day to Monday, and so on.
+        yearly_start (int): specifying the start day of the yearly seasonality plot.
+            0 (default) starts the year on Jan 1.
+            1 shifts by 1 day to Jan 2, and so on.
+        figsize (tuple): width, height in inches.
+            None (default):  automatic (10, 3 * npanel)
+
+    Returns:
+        A plotly figure.
+    """
+
+    parameter_components = get_parameter_components(m, forecast_in_focus)
+
+    components = parameter_components["components"]
+    additive_future_regressors = parameter_components["additive_future_regressors"]
+    additive_events = parameter_component["additive_events"]
+    multiplicative_future_regressors = parameter_components["multiplicative_future_regressors"]
+    multiplicative_events = parameter_components["multiplicative_events"]
+
+    npanel = len(components)
+    figsize = figsize if figsize else (10, 3 * npanel)
+
+    # Create Plotly subplot figure and add the components to it
+    fig = make_subplots(npanel, cols=1, print_grid=False)
+    fig["layout"].update(go.Layout(showlegend=False, width=figsize[0], height=figsize[1] * npanel))
+
+    if npanel == 1:
+        axes = [axes]
+
+    for i, comp in enumerate(components):
+        is_multiplicative = False
+        plot_name = comp["plot_name"].lower()
+        if plot_name.startswith("trend"):
+            if "change" in plot_name:
+                # plot_trend_change(m=m, ax=ax, plot_name=comp["plot_name"])
+                trace_object = plot_trend_change(m, plot_name=comp["plot_name"])
+            else:
+                # plot_trend(m=m, ax=ax, plot_name=comp["plot_name"])
+                trace_object = plot_trend(m, plot_name=comp["plot_name"])
+
+        elif plot_name.startswith("seasonality"):
+            name = comp["comp_name"]
+            if m.season_config.mode == "multiplicative":
+                is_multiplicative = True
+            if name.lower() == "weekly" or m.season_config.periods[name].period == 7:
+                trace_object = plot_weekly(
+                    m=m, weekly_start=weekly_start, comp_name=name, multiplicative=is_multiplicative
+                )
+            elif name.lower() == "yearly" or m.season_config.periods[name].period == 365.25:
+                trace_object = plot_yearly(
+                    m=m, yearly_start=yearly_start, comp_name=name, multiplicative=is_multiplicative
+                )
+            elif name.lower() == "daily" or m.season_config.periods[name].period == 1:
+                trace_object = plot_daily(m=m, comp_name=name, multiplicative=is_multiplicative)
+            else:
+                trace_object = plot_custom_season(m=m, ax=ax, comp_name=name, multiplicative=is_multiplicative)
+        elif plot_name == "lagged weights":
+            trace_object = plot_lagged_weights(
+                weights=comp["weights"], comp_name=comp["comp_name"], focus=comp["focus"], ax=ax
```

Comment:
```suggestion
                weights=comp["weights"], comp_name=comp["comp_name"], focus=comp["focus"]
```
Remove unnecessary ax=ax